### ISSUE TITLE
Does not work on Wayland

### ISSUE THREAD

**Issue by Flimm** [stuartlangridge/utm_no#6]
I tried running `python3 -m utm_no` on Ubuntu 24.04 with Wayland, and it doesn't seem to work at all. However, if I log in again, choosing to use Xorg instead of Wayland, then utm_no does succeed in removing the tracking parameters from URLs.

**Comment by stuartlangridge** [comment]
Yup. This monitors the X clipboard. I don't have Wayland, and I don't know how (or even if it's possible) to do what utm_no does under Wayland, I'm afraid!

**Comment by Flimm** [comment]
Ubuntu 26.04 LTS has just been released. It is no longer possible to run X11 with GNOME on Ubuntu 26.04.

**Comment by stuartlangridge** [comment]
I'm still in the position of not knowing at all how to do this under Wayland or even if it's possible. i don't run Wayland myself, so I can't really test it.
